pandas/tests/frame/test_constructors.py

```diff
@@ -2153,6 +2153,64 @@ def test_constructor_list_str_na(self, string_dtype):
         expected = DataFrame({"A": ["1.0", "2.0", None]}, dtype=object)
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("copy", [False, True])
```

Comment:
test_constructors.py should be in pandas/tests/frame/constructors, 

also this should be in pandas/tests/frame/constructors/test_from_dict no?

Reply:
thats specific to the DataFrame.from_dict method (there's a prominent comment in that file to that effect)

but you're right more generally that this file needs to be cleaned/split